modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -527,3 +529,31 @@ function editConsentStatusFields($db, $user)
         }
     }
 }
+
+/**
+ * Handles the updating of candidate's date of birth.
+ *
+ * @param Database $db   database object
+ * @param User     $user user object
+ *
+ * @throws DatabaseException
+ *
+ * @return void
+ */
+function editCandidateDOB($db, $user)
+{
+    if (!$user->hasPermission('candidate_parameter_edit')
```

Comment:
```suggestion
    if (!$user->hasAnyPermission(array('candidate_parameter_edit', 'candidate_dob_edit')))
```

If you want to, you can use this function to check multiple permissions.